File: pset6/mario/less/mario.py
import cs50

# print ("x", end="")

while True:
    height = cs50.get_int("Height: ")
    if (height >= 1 and height <= 8):
        break

# height = height + 1

for i in range(height):
    for k in range(height - i, 1, -1):
        print(" ", end="")
    for j in range(0, i + 1, 1):
        print("#", end="")
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    // if input not between 1 & 8 redo the above
    while (height < 1 || height > 8);
    
    // loop for new line
    for (int i = 0; i < height; i++)
    {
        // loop for spaces 
        for (int k = height - i; k > 1; k--)
        {
            printf(" ");
        }
        // loop for #'s
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

